packages/rancher-istio-1.9/charts/values.yaml

```diff
@@ -65,8 +65,6 @@ kiali:
     strategy: anonymous
   deployment:
     ingress_enabled: false
-    repository: rancher/mirrored-kiali-kiali
-    tag: v1.32.0
```

Comment:
fyi @nickgerace, brenda and I discussed this offline and we are removing the tag here since it is already tracked in the subchart's values.yaml.

Reply:
Makes sense to me! That's what I had thought, so thank you both for checking up on that.

Reply:
Was this not there for a reason before, not as a default or anything, it just got overridden immediately ?

Reply:
From what I remember, we had these there to ensure it was picked up for airgap image list, but we are handling that differently now. These are the same values that are being set in the subchart so there is no reason to override it here.